tests/commands_test.go

```diff
@@ -0,0 +1,58 @@
+package tests
+
+import (
+	"fmt"
+	"strings"
+	"testing"
+
+	"gotest.tools/v3/assert"
+)
+
+func TestCommandLIST(t *testing.T) {
+	commands := []string{
+		"ABORT", "BFADD", "BFEXISTS", "BFINFO", "BFINIT", "BGREWRITEAOF", "CLIENT",
+		"DEL", "DISCARD", "EXEC", "EXPIRE", "GET", "INCR", "INFO", "LATENCY", "LIST", "LRU",
+		"MULTI", "PING", "QINTINS", "QINTLEN", "QINTPEEK", "QINTREM", "QREFINS", "QREFLEN",
+		"QREFPEEK", "QREFREM", "QWATCH", "SET", "SLEEP", "STACKINTLEN", "STACKINTPEEK",
+		"STACKINTPOP", "STACKINTPUSH", "STACKREFLEN", "STACKREFPEEK", "STACKREFPOP",
+		"STACKREFPUSH", "SUBSCRIBE", "TTL",
+	}
+
+	subscriber := getLocalConnection()
+
+	rp := fireCommandAndGetRESPParser(subscriber, "LIST")
```

Comment:
No need to use `fireCommandAndGetRESPParser` here, you can directly use the `fireCommand` utility.

`fireCommandAndGetRESPParser` is used for commands that may return more data over time (eg. QWATCH)

Reply:
Interesting. Thanks